components/builder-depot/src/doctor.rs

```diff
@@ -154,7 +153,6 @@ impl<'a> Doctor<'a> {
 
     fn run(mut self) -> Result<Report> {
         try!(self.init_fs());
-        try!(self.truncate_datastore(&self.depot.datastore));
```

Comment:
@mwrock `doctor` should be moved to the `originsrv` and be a separate binary that we output. I think it's completely alright for now to break it in anyway and just revisit it. The work that @nellshamrell is doing right now to migrate data from an older builder cluster to a new one will most likely be re-usable in places to fill in what a proper `repair`/`doctor` command would look like